### PR TITLE
Stops experimentor from stealing Ian

### PR DESCRIPTION
**Pull request by Marmio64**
very very shitcode

#### Changelog

:cl:  
rscadd: Experimentor can't steal Ian anymore. Only Runtime. Fuck Runtime though.
/:cl:

### CODE REVIEW

**Comment by Mesalikepie** [comment]
Ian is a traitor objective. it's physically impossible to stop Ian being kidnapped with this shit. Speed merge plz

**Comment by alao3alao3** [comment]
We ought to have a PR broadly deny station pets or simplemobs of the like being stolen, rather than having once niche scenario boot a new dog out (as big of a meme it is), so long as the pr is anything more than spur of the moment.

File: code/modules/research/experimentor.dm [hunk 1]
```diff
@@ -533,13 +533,8 @@
...
 		throwSmoke(loc)
-		if(trackedIan)
-			throwSmoke(trackedIan.loc)
-			trackedIan.forceMove(loc)
-			investigate_log("Experimentor has stolen Ian!", INVESTIGATE_EXPERIMENTOR) //...if anyone ever fixes it...
-		else
-			new /mob/living/simple_animal/pet/dog/corgi(loc)
-			investigate_log("Experimentor has spawned a new corgi.", INVESTIGATE_EXPERIMENTOR)
+		new /mob/living/simple_animal/pet/dog/corgi(loc)
+		investigate_log("Experimentor has spawned a fake Ian.", INVESTIGATE_EXPERIMENTOR)
```
Comment (Hopekz) [inline code/modules/research/experimentor.dm]:
```suggestion
		investigate_log("Experimentor has spawned a corgi.", INVESTIGATE_EXPERIMENTOR)
```
No longer a fake Ian unless you add some code to rename the spawned corgi to Ian.

**Review by Hopekz — CHANGES_REQUESTED** [review]

**Comment by alao3alao3** [comment]
shit they really are gonna merge this aren’t they

File: code/modules/research/experimentor.dm (hunk 1 above)
Comment (Altoids1) [reply code/modules/research/experimentor.dm]: An alternative would be to make this Ian's evil twin or something, like a Star Trek teleporter-accident episode.

**Comment by Marmio64** [comment]
maybe another PR can be done in the future to make the spawned corgi be an evil ian twin or something, for now it will be a generic corgi.

**Comment by Moggram** [comment]
> 
> 
> maybe another PR can be done in the future to make the spawned corgi be an evil ian twin or something, for now it will be a generic corgi.

But only if he has  a goatee beard.

**Comment by redmoogle** [comment]
New piano type is /obj/structure/musician/piano

**Comment by alao3alao3** [comment]
But how will this stop people genuinely considering this pr?

**Review by Amelia0010 — APPROVED** [review]
Fair enough really

**Pull request merged**